cmd/dlv/cmds/commands.go

```diff
@@ -138,21 +138,48 @@ func New(docCall bool) *cobra.Command {
 	}
 
 	rootCommand.PersistentFlags().StringVarP(&addr, "listen", "l", "127.0.0.1:0", "Debugging server listen address. Prefix with 'unix:' to use a unix domain socket.")
+	if err := rootCommand.RegisterFlagCompletionFunc("listen", cobra.NoFileCompletions); err != nil {
```

Comment:
add 

```
func must(err error) {
    if err != nil {
        log.Fatal(err)
    }
}
```

and wrap all of this calls with it (or we could also not check the error).

Reply:
Done. Would rather check the error as the failure cases are delve side programming errors.